backend/src/web-server/routes/index.ts

```diff
@@ -94,6 +94,13 @@ router.post('/api/north/:northId/subscriptions/:southId', (ctx: KoaContext<any,
 router.delete('/api/north/:northId/subscriptions/:southId', (ctx: KoaContext<any, any>) =>
   subscriptionController.deleteNorthSubscription(ctx)
 );
+router.get('/api/north/:northId/external-subscriptions', (ctx: KoaContext<any, any>) => subscriptionController.getExternalNorthSubscriptions(ctx));
+router.post('/api/north/:northId/external-subscriptions/:southName', (ctx: KoaContext<any, any>) =>
```

Comment:
To have some consistency, southName should be renamed reference: it is not always sent by a south, but it can be sent directly from another application through the OIBus API, so the reference field refers to this source (another OIBus or another app)